assets/js/chooser.js

```diff
@@ -169,6 +169,18 @@ $(document).on("rendered", function() {
         }
     });
 
+    // Parse and set language from anchor tag if present.
+    var url = window.location.href;
```

Comment:
Could we use [`window.location.hash`](https://developer.mozilla.org/en-US/docs/Web/API/Location/hash) instead here? It would allow you to get rid of the next line where you are extracting the hash fragment from the `href`.

Reply:
Sure. Didn't even know that was a thing. Thanks.